wagtail/admin/templates/wagtailadmin/page_privacy/no_privacy.html

```diff
@@ -0,0 +1,5 @@
+{% load i18n %}
+{% include "wagtailadmin/shared/header.html" with title=_("Page privacy") icon="no-view" %}
+<div class="nice-padding">
+    <p>{% trans "Changing the privacy for this page has been disabled." %}</p>
+</div>
```

Comment:
Thanks, you can remove the header and padding to avoid the same issue as #11984

```suggestion
<p>{% trans "Changing the privacy for this page has been disabled." %}</p>
```